src/containers/RedashContainer/index.jsx

```diff
@@ -39,12 +37,11 @@ const telemetryDataToDatasets = (data, dataKeyIdentifier) => {
 
     return result;
   }, {});
-  const datasets = Object.keys(buckets).map((key, index) => {
+  const datasets = Object.keys(buckets).map(key => {
```

Comment:
As above.

Reply:
same reason, default styling is done inside the component, rather then at call time.